packages/docs-site/src/content/docs/guides/app-developers/deploy-a-contract.mdx

```diff
@@ -81,7 +81,7 @@ Hardhat is an Ethereum development environment for deploying smart contracts, ru
     ```
     - Initialize Hardhat
     ```bash
-    npx hardhat init
+    npx hardhat
```

Comment:
`npx hardhat init` is correct for hardhat v2. For hardhat v3 it is `npx hardhat --init`. Just `npx hardhat` is invalid.